entrypoint.sh

```diff
@@ -222,7 +222,7 @@ export COQ_IMAGE
 
 ## Note to docker-coq-action maintainers: Run ./helper.sh gen & Copy min.sh
 # shellcheck disable=SC2046,SC2086
-docker run --pull=never -i --init --rm --name=COQ $( [ -n "$INPUT_EXPORT" ] && printf -- "-e %s " $INPUT_EXPORT ) -e WORKDIR="$WORKDIR" -e PACKAGE="$PACKAGE" \
+docker run --pull=never -i --init --rm --name=COQ $( [ -n "$INPUT_EXPORT" ] && printf -- "-e %s " $INPUT_EXPORT ) -e GITHUB_ENV -e GITHUB_OUTPUT -e GITHUB_STEP_SUMMARY -e WORKDIR="$WORKDIR" -e PACKAGE="$PACKAGE" \
```

Comment:
Do we want to export more?  There's a lot of variables at https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
I don't think so. At least not for the github-runner-file-commands:
the leftover variable GITHUB_PATH does not make sense because the docker-coq-action and the next step have disjoint filesystems.

I don't mean extending this list further would be useless, but I think that this is beyond the scope of this PR.